pkg/scheduler/scheduler.go

```diff
@@ -610,7 +607,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 	// Synchronously attempt to find a fit for the pod.
 	start := time.Now()
 	state := framework.NewCycleState()
-	state.SetRecordFrameworkMetrics(rand.Intn(100) < frameworkMetricsSamplePercent)
+	state.SetRecordFrameworkMetrics(true)
```

Comment:
This will always record plugin metrics, which we don't want - some plugins run per pod and it's very expensive to always record.